cpp/oneapi/dal/algo/jaccard/backend/cpu/vertex_similarity_default_kernel_cpu.cpp

```diff
@@ -26,26 +25,13 @@
 
 namespace oneapi::dal::preview {
```

Comment:
Probably namespaces should be in one line

Reply:
updated.